addons/disable-sprite-wobble/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Disable sprite wobble in sprite pane",
-  "description": "Allow scripts, costumes, and sounds to be moved to different sprites without the sprites moving around within the sprite pane.",
+  "name": "Fix sprite pane glitching",
+  "description": "Allow scripts, costumes, and sounds to be dragged and dropped onto different sprites without the sprites jumping around within the sprite pane.",
```

Comment:
Maybe the description should start with `Fixes a bug in the Scratch editor where...` or `Fixes a Scratch bug that...` just like other bug fix addons

Reply:
Also, is it "drag and drop into" or "drag and drop onto"?

Reply:
into I think